### ISSUE TITLE
The provided secret has expired when selecting view

### ISSUE THREAD

**Issue by dottodot** [chapter-three/next-drupal#422]
I've noticed an issue recently where pressing the view button results in the site loading the api/preview route with the The provided secret has expired response. I believe this happens if you press view then trigger the exit-preview route reload the page then go back and press view again.

Does the view button need to go via the api/preview route? I would have thought it could just go directly to the page url.

**Comment by shadcn** [comment]
Yes. This is a known "issue". We're looking into a fix where the button requests a new preview URL when clicked.

<img width="1020" alt="fGk4bVv8" src="https://user-images.githubusercontent.com/124599/213479363-96c1c450-5288-410d-b0da-e4a30d712d5a.png">

For now, the workaround is to refresh the iframe page to request a new link

**Comment by AlexandreBourdeaudhui** [comment]
Hello, we have the same issue here, with a preview of content.
When we're pushing the "view" button, after the 30 seconds, we got a message `The provided secret has expired.`. We need to refresh the iframe to get the "View" button to get working again.
